client-tests.js

```diff
@@ -1,16 +1,3 @@
-Tinytest.add('aws-sdk', function (test) {
-  var isDefined = false;
-  try {
-    AWS;
-    isDefined = true;
-  }
-  catch (e) {
-  }
-  test.isTrue(isDefined, "AWS is not defined");
-  test.isTrue(Package['peerlibrary:aws-sdk'].AWS, "Package.peerlibrary:aws-sdk.AWS is not defined");
-
-  var s3 = new AWS.S3();
```

Comment:
Could you keep these tests?

Reply:
For lazy loading it shouldn't get imported. That's why i have removed these test. Also i was unable to find out how to test out lazy loading package. Please guide me.